mpm_config

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Opens the mpm coins config file for editing.
+#
+# @author lightspeed393
+# @link https://github.com/cipig/mmtools
+#
+if [[ -z $1 ]]; then
+    echo "You must specify a base coin to configure. (BTC, KMD, BCH, LTC, or DOGE)"
+    echo ""
+    echo "./mpm_config KMD"
+    exit 0
+fi
+
+if [[ $1 == "BTC" ]] || [[ $1 == "KMD" ]] || [[ $1 == "BCH" ]] || [[ $1 == "LTC" ]] || [[ $1 == "DOGE" ]]; then
+    nano mpm/mpm.conf.dex.$1
+else
+    echo "Choose from BTC, KMD, BCH, LTC or DOGE."
+fi
```

Comment:
This feels strange to a file to run nano :)

Reply:
Yes I suppose. My only goal was to make things as basic as possible that any user could theoretically do it. Happy to edit/delete whatever you feel is appropriate.

Reply:
I guess this is ok. Maybe need to add `nano` in `init` for the meantime.

Reply:
I added `nano` dependency  to `init`.